src/mscorlib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -17,5 +18,11 @@ public static class EnvironmentAugments
         public static bool HasShutdownStarted => Environment.HasShutdownStarted;
         public static string StackTrace => Environment.StackTrace;
         public static int TickCount => Environment.TickCount;
+        public static string GetEnvironmentVariable(string variable) => Environment.GetEnvironmentVariableCore(variable);
```

Comment:
> differs between Windows and Unix.

Apparently I confused your request on the Windows code. Why do you want only the registry code to be in CoreFX?